packages/earljs/src/test-runners/uvu.ts

```diff
@@ -0,0 +1,65 @@
+import debug from 'debug'
+import { Callback, Test } from 'uvu'
+
+import { setTestRunnerIntegration } from '../testRunnerCtx'
+import { TestRunnerCtx } from './TestRunnerCtx'
+
+const d = debug('earljs:uvu')
+
+function main() {
+  if (typeof require !== 'function') {
+    return;
+  }
+
+  d('earljs/uvu integration is being registered...')
+
+  const uvu: typeof import('uvu') = require('uvu')
```

Comment:
We need to monkey patch the uvu module to hook into the test executions, but we can't search the `require` cache like the mocha integration, since uvu hasn't been required yet at this point. I'm not sure if this is a problem or not?